source/frontend/src/features/mapSideBar/management/tabs/activities/edit/ManagementActivityEditContainer.tsx

```diff
@@ -61,17 +62,31 @@ export const ManagementActivityEditContainer: React.FunctionComponent<
         }
         await fetchProviderContact(retrieved);
 
-        setInitialValues(ManagementActivityFormModel.fromApi(retrieved));
+        setInitialValues(ManagementActivityFormModel.fromApi(retrieved, castedFile.fileProperties));
       }
     } else {
       // Create activity flow
       const defaultModel = new ManagementActivityFormModel(null, managementFileId);
       defaultModel.activityStatusCode = 'NOTSTARTED';
       defaultModel.requestedDate = getCurrentIsoDate();
+      defaultModel.selectedProperties = castedFile.fileProperties.map(x => {
+        return {
+          id: x.id,
+          fileId: castedFile.id,
+          propertyName: x.propertyName,
+          location: x.location,
+          displayOrder: x.displayOrder,
+          property: x.property,
+          propertyId: x.propertyId,
+        } as ApiGen_Concepts_FileProperty;
+      });
```

Comment:
I like this approach but have a couple of minor comments. I'd like to see a null check before using `castedFile.fileProperties` to defend against NPE. Something like `(castedFile.fileProperties ?? []).map(...)`

The second comment is why we have to "force" the typing with "as ApiGen_Concepts_FileProperty;" - we usually do that when there are additional properties we don't want to populate but they are present in the type. 
If you specified all the properties then I don't think we need the "as ..." - thoughts?